### ISSUE TITLE
[Meeting] Inventory Check-in 7/27

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#6785]
### Objective 
Checking up

### Participants
Ray, Ronnie, Frank, Johnny, John, Jorge G.

### Agenda
Add agenda here or create agenda from [this template](https://docs.google.com/document/d/1d_49KW5C_vSz8Bs50v-cxyIJuTNJMwMrh7ypcxRHgZI/edit#) and add link. 

------
- [x] Schedule meeting
- [ ] Optional: Schedule debrief
- [x] Meet and take notes
- [ ] Optional: Debrief with DTS team members
- [ ] Create resulting issues

**Comment by dianamartin** [comment]
# Diana's Notes

## Vendor Issue - Packing Slips
- Difficult to reference commodity code 
- Asked vendor to use part number and description
- Packing slips aren't really descriptive
- So it's easy to cross reference the receipts
- Placed an order for poles & arms, Ray asked vendor to put down pole type

## Reconciliation Oddities

**Comment by dianamartin** [comment]
John replied 7/27
>Following up on TXN ID `10518`. Here's the inventory request.

This is the case of a final unit cost not matching the unit cost of the item that was issued. This occurred on 5/20. We made a change in the Data Tracker in early June that prevents this situation from happening. We did this in response to an identical issue that happened around the same time. I just re-tested that functionality, and it is working correctly.

I will run through the transactions and make sure we don't have any more like this and will work with Van to correct the booking.
